### DISCUSSION TITLE
Ability to Assign Values Using Knobs

### DISCUSSION THREAD

**Discussion by Dr3wPdraw3rs** [cpyarger/obs-midi#99] [Ideas]
Would love for the ability to assign numerical values to obs filter settings using knobs (Turn up and down opacity, change saturation, etc using a midi knob)

This control ability is already utilized for the "Set Volume" action, would just love more use out of this control type.

The work is awesome!

Thank You,
DrewPdrawers

**Comment by cpyarger** [comment]
You can now rotate sources (: working on figuring out more

**Reply by Dr3wPdraw3rs** [reply]
That's awesome!

Thank you for all of your amazing work!

**Comment by diotro** [comment]
If you could just allow the user to map a control change knob to a filter's slider and adjust parameters on a logarithmic scale 0-127 that would be awesome for live filter adjustments for a more artsy fartsy type of stream!

https://user-images.githubusercontent.com/5402413/136682723-cf17f655-1385-4c25-927b-719bb7d009cc.mp4
